File: src/main/java/me/lovesasuna/lanzou/utils/ReaderUtil.java
package me.lovesasuna.lanzou.utils;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.Reader;

/**
 * @author LovesAsuna
 * @date 2020/8/22 12:44
 **/
public class ReaderUtil {
    public static void readAnyTime(int times, BufferedReader reader) {
        BasicUtil.runAnyTimes(times,reader,r -> {
            try {
                r.readLine();
            } catch (IOException e) {
                e.printStackTrace();
            }
        });
    }
}

File: src/main/kotlin/me/lovesasuna/lanzou/util/ReaderUtil.kt
package me.lovesasuna.lanzou.util

import java.io.BufferedReader
import java.io.IOException

/**
 * @author LovesAsuna
 * @date 2020/8/22 12:44
 */
object ReaderUtil {
    fun readAnyTime(times: Int, reader: BufferedReader) {
        BasicUtil.runAnyTimes(times, reader, { r: BufferedReader ->
            try {
                r.readLine()
            } catch (e: IOException) {
                e.printStackTrace()
            }
        })
    }
}